pbench_logparse/main.go

```diff
@@ -0,0 +1,317 @@
+package main
+
+import (
+	"bufio"
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/sjug/go-logparse/stats"
+)
+
+var searchDir, resultDir, processes string
+var processList []string
+var blockDevices, netDevices stringSlice
+var fileHeader = map[string][]string{}
+
+type stringSlice []string
```

Comment:
Ok, so my question is why do we want to create a custom flag in first place? If we have comma separated value, we could easily pass them around as slice.